### PR TITLE
When constructing ChowChow object, pass in ChowOptions

### PR DESCRIPTION
**Pull request by crizo23**
The ChowChow constructor takes an optional ChowOptions. This will be valuable when custom AJV validation is needed.

Fixes https://github.com/atlassian/koa-oas3/issues/22

### CODE REVIEW

**Comment by crizo23** [comment]
Anyone know what this build error is? "Workflows can't be executed on this repository. Please check your payment method or billing status."

**Comment by supertong** [comment]
@crizo23 I think we don't pay for Action for Github and that's why it fails. I will remove it.

**Comment by supertong** [comment]
Hey @crizo23 , would you be able to re-generate the version using `yarn changeset` command?

https://github.com/atlassian/changesets/blob/master/packages/cli/README.md

**Comment by crizo23** [comment]
> Hey @crizo23 , would you be able to re-generate the version using `yarn changeset` command?
> 
> https://github.com/atlassian/changesets/blob/master/packages/cli/README.md

Thanks, I just used yard changeset and committed. Please let me know if I didn't use changeset correctly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 # koa-oas3
 
+## 0.16.0
```
Comment (supertong) [inline CHANGELOG.md]: We don't need to manually update this CHANGELOG. It will be automatically populated during the release.

Comment (supertong) [reply CHANGELOG.md]: Hey @crizo23 would you be able to remove the changes of this file from the commit?

Comment (crizo23) [reply CHANGELOG.md]: Fixed, thanks @supertong

**Pull request merged**